vanilla-framework/vanilla-framework
# 1490: Add vendor prefixes to form elements

## Done

- Added vendor prefixes to form elements, to help with projects that don't have autoprefixer

## QA

- Pull code
- Remove all mentions of gulp autoprefixer from vanilla
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/forms/selects/
- Check that the select element does not change appearance

## Details

Fixes #1490 

## Screenshots

### Before:
![screenshot 8](https://user-images.githubusercontent.com/25733845/35329228-ffafe44e-00f6-11e8-835c-fe48ee9ed21e.png)

### After:
![screenshot 9](https://user-images.githubusercontent.com/25733845/35329237-0c433e04-00f7-11e8-9daf-9070b019437b.png)
